src/api/statements/JSONLDObject/mergeClasses.js

```diff
@@ -1,3 +1,4 @@
+import { host } from 'config';
```

Comment:
Oops. This should not be done in `statements`, as they are aimed to be exported to another npm package: `ezs-lodex`.

Reply:
Note that exist this statement : https://github.com/Inist-CNRS/lodex/blob/master/src/api/statements/LodexConfig.js

so we can not import config. We just need to test if input data contains $context. like thishttps://github.com/Inist-CNRS/lodex/blob/master/src/api/statements/LodexSetField.js#L14